app/models/health/claims/view_model.rb

```diff
@@ -19,73 +18,75 @@ def initialize(patient, sdh_rosters)
       if @patient_roster.present?
         values = load_implementation_baseline_variance(load_cost_values)
         values.each_with_index do |v, index|
           key = result.keys[index]
           result[key] = {
             Total_Cost: display_value(key, v[0], number_to_currency(v[0], precision: 0)),
             Months: display_value(key, v[1], v[1]),
-            Cost_PMPM: display_value(key, v[2], number_to_currency(v[2], precision: 0))
+            Cost_PMPM: display_value(key, v[2], number_to_currency(v[2], precision: 0)),
           }
         end
       end
       load_table(result)
     end
 
     def key_metrics_table
-      result = {patient: {}, sdh: {}, variance: {}}
+      result = { patient: {}, sdh: {}, variance: {} }
       if @patient_roster.present?
         values = load_implementation_baseline_variance(load_key_metric_values)
         values.each_with_index do |v, index|
           key = result.keys[index]
           result[key] = {
             ED_Visits: display_value(key, v[0], v[0]),
             IP_Admits: display_value(key, v[1], v[1]),
-            Average_Days_to_Readmit: display_value(key, v[2], v[2])
+            Average_Days_to_Readmit: display_value(key, v[2], v[2]),
           }
         end
       end
       load_table(result)
     end
 
     def patient_summary
-      result = {details: [], demographics: []}
+      result = { details: [], demographics: [] }
       if @patient.present?
         if @patient_roster.present?
-          team = @patient_roster.epic_team
           disability_flag = @patient_roster.disability_flag ? 'Y' : 'N'
         else
-          team = 'Unknown'
           disability_flag = 'Unknown'
         end
         result[:details] = [
           ['SSN', @patient.ssn],
           ['Medicaid ID', @patient.medicaid_id],
           ['Primary Care Provider', @patient.primary_care_physician],
           ['Assignment Date', @patient.enrollment_start_date],
-          # ['Team', team]
         ]
-        result[:demographics].push([
-          ['Age', @patient.client.age],
-          ['Gender', @patient.gender],
-          ['Disability Flag', disability_flag],
-        ])
-        result[:demographics].push([
-          ['DOB', @patient.birthdate],
-          ['Race / Ethnicity', "#{@patient.race} / #{@patient.ethnicity}"],
-          ['Veteran Status', @patient.veteran_status]
-        ])
+        result[:demographics].push(
+          [
+            ['Age', @patient.client.age],
+            ['Gender', @patient.gender],
+            ['Disability Flag', disability_flag],
+          ],
+        )
+        result[:demographics].push(
+          [
+            ['DOB', @patient.birthdate],
+            ['Race / Ethnicity', "#{@patient.race} / #{@patient.ethnicity}"],
+            ['Veteran Status', @patient.veteran_status],
+          ],
+        )
       end
       result
     end
 
     protected
 
+    Table = Struct.new('Table', :keys, :patient, :sdh, :variance)
```

Comment:
How about a comment about what's going on here.  It looks like you're essentially defining a new class:
`Health::Claims::ViewModel::Table` but in an odd way.  Why not?
```
class Table < Struct
end
```
or something similar?

Reply:
I replaced this with an OpenStruct -- while they are a little slower, the purpose here is just passing a structured object for downstream readability.